runtime/src/impls.rs

```diff
@@ -309,10 +360,56 @@ impl GetDispatchInfo for DispatchableSubstrateBridgeCall {
                 let call: multisig_verifier::Call<crate::Runtime> = msg.clone().into();
                 call.get_dispatch_info()
             }
+            bridge_types::substrate::BridgeCall::SubstrateApp(_) => {
```

Comment:
I think it's better to place `wip` feature flag condition here to not have code duplication